src/index.html

```diff
@@ -1,14 +1,147 @@
+    <p>
+      No one shall be held in slavery or servitude; slavery and the slave trade shall be prohibited in all their forms.
+    </p>
+
+    <h2>Article 5</h2>
+    <p>
+      No one shall be subjected to torture or to cruel, inhuman or degrading treatment or punishment.
+    </p>
+
+    <h2>Article 6</h2>
+    <p>
+      Everyone has the right to recognition everywhere as a person before the law.
+    </p>
+
+    <h2>Article 7</h2>
+    <p>
+      All are equal before the law and are entitled without any discrimination to equal protection of the law. All are entitled to equal protection against any discrimination in violation of this Declaration and against any incitement to such discrimination.
+    </p>
+
+    <h2>Article 8</h2>
+    <p>
+      Everyone has the right to an effective remedy by the competent national tribunals for acts violating the fundamental rights granted him by the constitution or by law.
+    </p>
+
+    <h2>Article 9</h2>
+    <p>
+      No one shall be subjected to arbitrary arrest, detention or exile.
+    </p>
+
+    <h2>Article 10</h2>
+    <p>
+      Everyone is entitled in full equality to a fair and public hearing by an independent and impartial tribunal, in the determination of his rights and obligations and of any criminal charge against him.
+    </p>
+
+    <h2>Article 11</h2>
+    <ol>
+      <li>Everyone charged with a penal offence has the right to be presumed innocent until proved guilty according to law in a public trial at which he has had all the guarantees necessary for his defence.
+      </li>
+      <li>
+        No one shall be held guilty of any penal offence on account of any act or omission which did not constitute a penal offence, under national or international law, at the time when it was committed. Nor shall a heavier penalty be imposed than the one that was applicable at the time the penal offence was committed.
+      </li>
+    </ol>
+
+    <h2>Article 12</h2>
+    <p>
+      No one shall be subjected to arbitrary interference with his privacy, family, home or correspondence, nor to attacks upon his honour and reputation. Everyone has the right to the protection of the law against such interference or attacks.
+    </p>
+
+    <h2>Article 13</h2>
+    <ol>
+      <li> Everyone has the right to freedom of movement and residence within the borders of each State.
+      </li>
+      <li>
+        Everyone has the right to leave any country, including his own, and to return to his country.
+      </li>
+    </ol>
+
+    <h2>Article 14</h2>
+    <ol>
+      <li>
+        Everyone has the right to seek and to enjoy in other countries asylum from persecution.
+      </li>
+      <li>
+        This right may not be invoked in the case of prosecutions genuinely arising from non-political crimes or from acts contrary to the purposes and principles of the United Nations.
+      </li>
+    </ol>
+
+    <h2>Article 15</h2>
+    <ol>
+      <li>Everyone has the right to a nationality.
+      </li>
```

Comment:
```suggestion
      <li>Everyone has the right to a nationality.</li>
```
If an element takes less than 1 line, there is no need to move closing tag on the next line

Reply:
same everywhere